contracts/ethereum-sc/contracts/ERC20Safe.sol

```diff
@@ -157,11 +159,12 @@ contract ERC20Safe {
     function getNextPendingBatch() public view returns (Batch memory) {
         Batch memory batch = batches[currentPendingBatch];
 
-        if((batch.timestamp + batchTimeLimit) < block.timestamp || batch.deposits.length >= batchSize)
+        if(((batch.timestamp + batchTimeLimit) < block.timestamp || batch.deposits.length >= batchSize) 
```

Comment:
In my proposal I suggested to completely remove this check and only take into account if the batch is settled time-wise.

Currently this check has 2 possible effects:
1. `batchTimeLimit <= batchSettleLimit` (as it is now) - when the batch is settled, this line will always be true because the `batchTimeLimit` has already passed too.
2. `batchTimeLimit > batchSettleLimit` - this leaves a window in which the same reorg issue is possible.

Or is there a reason I am missing why we kept this check? :)

Reply:
I implemented the change above in order to be able to merge this PR